### PR TITLE
fix(webpaas): fix storage translation

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-7136
| License          | BSD 3-Clause

## Description

fix storage translation to add included translations
fix missing coming soon translation

### CODE REVIEW

File: packages/manager/modules/web-paas/src/details/service/service.html [hunk 1]
```diff
@@ -93,7 +93,8 @@
                     <oui-tile-description>
                         <p data-ng-if="$ctrl.project.addonStorageCount() > 0">
                             <span
-                                data-ng-bind="$ctrl.project.getTotalStorage()"
+                                data-translate="web_paas_service_project_storage_included"
+                                data-translate-values="{storage: $ctrl.project.getTotalStorage() - ($ctrl.project.addonStorageCount() * $ctrl.STORAGE_MULTIPLE)}"
```
Comment (JeremyDec) [inline packages/manager/modules/web-paas/src/details/service/service.html:97]: This logic should be done into the controller

Comment (varun257) [reply packages/manager/modules/web-paas/src/details/service/service.html]: Has been donen

File: packages/manager/modules/web-paas/src/details/service/translations/Messages_es_ES.json [hunk 2]
```diff
@@ -37,5 +37,6 @@
   "web_paas_expiry_date": "Fecha de expiración",
   "web_paas_tile_subscription_manage_service": "Gestionar el servicio",
   "web_paas_tile_subscription_pay_billing": "Pagar la factura",
-  "web_paas_tile_billing_error": "Se ha producido un error al redirigir la página a la facturación: "
+  "web_paas_tile_billing_error": "Se ha producido un error al redirigir la página a la facturación: ",
+  "web_paas_service_project_coming_soon": "Próximamente disponible"
```
Comment (frenautvh) [inline packages/manager/modules/web-paas/src/details/service/translations/Messages_es_ES.json:41]: where does those translations comes from ? i don't see the translator commit, did you add them manually ?

Comment (varun257) [reply packages/manager/modules/web-paas/src/details/service/translations/Messages_es_ES.json:41]: @frenautvh 
We had this translation already done but i lost it when i rebased. So i added it manually.

File: packages/manager/apps/web-paas/package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-web-paas-app' --include-dependencies -- npm run dev:watch --if-present"
   },
   "dependencies": {
+    "@ovh-ux/manager-billing": "^1.0.5 || ^2.0.0",
```
Comment (frenautvh) [inline packages/manager/apps/web-paas/package.json:21]:
```suggestion
    "@ovh-ux/manager-billing-components": "^2.0.0 || ^3.0.0",
```

this module just have been renamed

**Review by frenautvh — CHANGES_REQUESTED** [review]

**Review by radireddy — CHANGES_REQUESTED** [review]

**Comment by marie-j** [comment]
It looks like some commit should be splitted. Indeed first commit is `fix(webpaas): fix storage translation` but also fixes a dependency

File: packages/manager/modules/web-paas/src/details/service/translations/Messages_es_ES.json (hunk 2 above)
Comment (varun257) [reply packages/manager/modules/web-paas/src/details/service/translations/Messages_es_ES.json]: @radireddy, Please check now, it looks clean and i have imported the new translation changes.

File: packages/manager/modules/web-paas/src/details/service/translations/Messages_en_GB.json [hunk 4]
```diff
@@ -37,5 +37,6 @@
   "web_paas_expiry_date": "Expiry date",
   "web_paas_tile_subscription_manage_service": "Manage service",
   "web_paas_tile_subscription_pay_billing": "Pay bill",
-  "web_paas_tile_billing_error": "An error has occurred redirecting to billing: "
+  "web_paas_tile_billing_error": "An error has occurred redirecting to billing: ",
+  "web_paas_service_project_coming_soon": "Coming soon"
```
Comment (ganeshkumar1989) [inline packages/manager/modules/web-paas/src/details/service/translations/Messages_en_GB.json]: Doesn't this require a translation request?

**Review by ganeshkumar1989 — APPROVED** [review]

**Comment by JeremyDec** [comment]
Hello @varun257, it seems that your first commit still does more than just fix the translations. Could the commit be split to have those changes on another commit ?

**Review by radireddy — APPROVED** [review]

**Comment by varun257** [comment]
@frenautvh , Can you review this please ?

**Review by antleblanc — APPROVED** [review]

**Pull request merged**